### PR TITLE
Joke Flask App pull Request

### PR DESCRIPTION
**Pull request by OnurSefa**
I have created an App in which a user may find funny jokes and mark them as favorites. After that, s/he may view their favorites joke in different page. It was fun also good for learning the base structure of API relationships in Flask technology.

### CODE REVIEW

**Comment by gkberk** [comment]
Job well done, thanks!

**Review by gkberk — APPROVED** [review]

**Review by safaaltunel — COMMENTED** [review]
Nice work done, mate! If you fix the parts that I mentioned, it is ready to go!

File: Joke_Flask_App/app.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+
+@app.route('/add_to_list/<user_name>/<set_up>/<punch_line>', methods=["GET", "PUT"])
+def add_to_list(user_name, set_up, punch_line):
+    if request.method == "PUT":
+        return helper_functions.add_to_list(user_name, set_up, punch_line, collection)
+    else:
+        return None
+
+
```
[the remark this answers is not in the document]
Comment (OnurSefa) [reply Joke_Flask_App/app.py:54]: I have added /joke base url address to the all relevant urls with new commit

**Review by ekremgezgen — APPROVED** [review]
Nice work. Thanks

**Pull request merged**